partition_tables_templates/kickstart_uefi_custom.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+kind: ptable
+name: Kickstart UEFI custom
+model: Ptable
+oses:
+- CentOS
+- Fedora
+- RedHat
+%>
+zerombr
+clearpart --all --initlabel
+<% if @host.pxe_loader.include?("UEFI") -%>
+  
+part /boot/efi --fstype="efi" --ondisk=sda --size=200 --fsoptions="umask=0077,shortname=winnt"
```

Comment:
Should sda be a parameter but with a default?

Reply:
Definitely, I wanted to do this not sure why I pushed this WIP version.